component/core/src/test/java/org/exoplatform/social/core/manager/IdentityManagerTest.java

```diff
@@ -589,6 +593,15 @@ public void aboutMeUpdated(ProfileLifeCycleEvent event) {
     identityManager.updateProfile(profile, true);
     assertEquals(3, changes.size());
     assertTrue(changes.contains(1));
+
+
+    InputStream inputStream = getClass().getResourceAsStream("/eXo-Social.png");
+    BannerAttachment bannerAttachment = new BannerAttachment(null, "banner", "png", inputStream, System.currentTimeMillis());
+    profile.setProperty(Profile.BANNER, bannerAttachment);
+    identityManager.updateProfile(profile, true);
+    assertEquals(5, changes.size());
```

Comment:
Why is there 5 changes instead of 4 ? the previous test asserts that there were 3 
Wht is the extra property changed ?

Reply:
resolved by creating a test dedicated to update banner